addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2557,11 +2557,21 @@ export class OdooEditor extends EventTarget {
      * @private
      */
     _onSelectionChange() {
+        const selection = this.document.getSelection();
+        // When CTRL+A in the editor, sometimes the browser use the
+        // editable element as an anchor & focus node
+        // this is an issue for the commands and the toolbar.
+        // So we need to "fix" the selection to be base on the editable children
+        if ( selection.anchorNode === this.editable && selection.focusNode === this.editable) {
+            const firstChild = this.editable.firstElementChild;
+            const lastChild = this.editable.lastElementChild;
+            setSelection(firstChild, 0, lastChild, lastChild.childNodes.length);
+            return;
```

Comment:
Why return in this case ? Because the `setSelection` will trigger its own onSelectionChange ?

Reply:
indeed

Reply:
I sugest to add a comment in order to make this explicit. :)